File: Add Code Here/C++/tree/MyFirstApplication/app/src/main/java/com/example/myfirstapplication/MainActivity.java
package com.example.myfirstapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.example.myfirstapplication.ui.login.LoginActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Intent callSignup=new Intent(getApplicationContext(), SignupActivity.class);
        startActivity(callSignup);
        System.out.println("I am here!");
    }
}

File: Add Code Here/Kotlin/savy/app/src/main/java/com/yash/savy/MainActivity.kt
package com.yash.savy

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val enter = findViewById<Button>(R.id.Enter)
        enter.setOnClickListener {
            val intent = Intent(this,HomeActivity::class.java)
            startActivity(intent)
        }
    }

}